### ISSUE TITLE
Invalid argument in UseGitVersion VS Task

### ISSUE THREAD

**Issue by bernard-arnaud** [GitTools/actions#40]
Hi,

We're having issues when using the UseGitVersion task on a dotnetcore 3.1 project.
Here are the logs :

```
Starting: UseGitVersion
==============================================================================
Task         : GitVersion Task
Description  : Easy Semantic Versioning (http://semver.org) for projects using Git
Version      : 5.1.2
Author       : GitVersion Contributors
Help         : See the [documentation](http://gitversion.readthedocs.org/en/latest/) for help
==============================================================================

--------------------------
Installing GitVersion.Tool version 5.1
--------------------------
/opt/hostedtoolcache/dncs/3.1.100/x64/dotnet tool install GitVersion.Tool --tool-path /home/vsts/work/_temp --version 5.1.2
You can invoke the tool using the following command: dotnet-gitversion
Tool 'gitversion.tool' (version '5.1.2') was successfully installed.
Caching tool: GitVersion.Tool 5.1.2 x64
##[error]Error: EINVAL: invalid argument, readlink '/opt/hostedtoolcache/dncs/3.1.100/x64/dotnet'
Finishing: UseGitVersion
```

And here's the yaml part of the task :

```yaml

  - task: UseGitVersion@5
    inputs:
      versionSpec: '5.1'
      updateAssemblyInfo: true
```

**Comment by arturcic** [comment]
@Breederveld , @dnperfors  I think the issue is coming from here https://github.com/GitTools/actions/blob/master/src/core/dotnet-tool.ts#L98, is this correct?

**Comment by Breederveld** [comment]
@arturcic I think you are right, this looks exactly how I imagined the code would do its thing

**Comment by arturcic** [comment]
if you can suggest a better implementation, a PR is welcomed

**Comment by dnperfors** [comment]
@arturcic, is that whole if statement still necessary? When I look for DOTNET_ROOT, I can’t see any references to it (although I didn’t look further than this action repo)

**Comment by arturcic** [comment]
The DOTNET_ROOT is an environment variable used by dotnet core to resolve the global tool location, in our case Gitversion

**Comment by Breederveld** [comment]
Created PR https://github.com/GitTools/actions/pull/41

**Comment by arturcic** [comment]
@bernard-arnaud @dnperfors @nloum can you confirm it's fixed? I f you use the azure pipeline extension please use [this ](https://marketplace.visualstudio.com/items?itemName=gittools.gittools)extension. If you already had it installed in the organization, please remove it before installation as it is caching the older one

**Comment by dnperfors** [comment]
@arturcic, works for me :)

**Comment by arturcic** [comment]
ok great, closing the issue. Closed by #41
